sgx/build/Cargo.toml

```
@@ -9,7 +9,7 @@ readme = "README.md"
 rust-version = "1.76.0"
 
 [dependencies]
-cc = "1.0"
+cc = "1.2.10"
```

similar to displaydoc and zeroize consider the following if possible
```suggestion
cc = "1"
```